ui/components/app/signature-request/signature-request.component.js

```diff
@@ -13,10 +13,8 @@ import {
 import { MetaMetricsEventCategory } from '../../../../shared/constants/metametrics';
 import SiteOrigin from '../../ui/site-origin';
 import Button from '../../ui/button';
-import Typography from '../../ui/typography/typography';
 import ContractDetailsModal from '../modals/contract-details-modal/contract-details-modal';
 import {
-  TypographyVariant,
   FontWeight,
   TextAlign,
   TextColor,
```

Comment:
Could you also please update the deprecated DISPLAY and BLOCK_SIZES in favour of enums

Reply:
updated